app/components/app_patient_search_result_card_component.rb

```diff
@@ -68,10 +82,34 @@ def call
   private
 
   def programme_outcome_tag
-    status = @patient.vaccination_status(programme: @programme).status
+    render_status_tag(:vaccination, :programme)
+  end
+
+  def consent_status_tag
+    render_status_tag(:consent, :consent)
+  end
+
+  def triage_status_tag
+    render_status_tag(:triage, :triage)
+  end
+
+  def render_status_tag(status_type, outcome)
+    status =
+      @patient.public_send(
+        "#{status_type}_status",
+        programme: @programme
+      ).status
     render AppProgrammeStatusTagsComponent.new(
              { @programme => status },
-             outcome: :programme
+             outcome: outcome
            )
   end
+
+  def display_triage_status?
+    status = @patient.triage_status(programme: @programme).status
+
+    return true if @triage_status.present?
+
+    status == "required"
```

Comment:
```suggestion
    @patient.triage_status(programme: @programme).required?
```

Reply:
That causes the tests to fail but we could do this:

```
return true if @triage_status.present?

@patient.triage_status(programme: @programme).required?
```